src/contexts/WalletContext/WalletContext.tsx

```diff
@@ -23,14 +23,24 @@ export default function WalletContext(props: PropsWithChildren<{}>) {
     "https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn"
   );
 
-  const binance = useInjectedProvider(
-    "binance-wallet",
+  const trust = useInjectedProvider(
```

Comment:
binance wallet is depracated and they are recommending to use trust wallet as replacement